ros_buildfarm/release_job.py

```diff
@@ -326,6 +326,10 @@ def _get_and_parse_distribution_cache(index, rosdistro_name, pkg_names):
     condition_context = get_package_condition_context(index, rosdistro_name)
     for pkg in cached_pkgs.values():
         pkg.evaluate_conditions(condition_context)
+    for pkg in cached_pkgs.values():
+        for group_depend in pkg.group_depends:
+            if group_depend.evaluated_condition is not False:
+                group_depend.extract_group_members(dist_cache.values())
```

Comment:
```suggestion
                group_depend.extract_group_members(cached_pkgs)
```

Reply:
Still needs `.values()`, I believe.

This is fallout from https://github.com/ros-infrastructure/ros_buildfarm/pull/719#discussion_r371547433 that I never fixed up after the rebase.